core/src/services/sled/backend.rs

```diff
@@ -77,9 +85,23 @@ impl Builder for SledBuilder {
                 .set_source(e)
         })?;
 
+        // use "default" tree if not set
+        let tree_name = match self.tree.take() {
+            Some(tree) => tree,
+            None => "default".into(),
```

Comment:
I'm not sure if I need to ensure overall compatibility here. Do I need to consider that the old version of the sled service does not have this tree configuration and therefore cannot use the default value directly?

Reply:
I think we need to use `sled::open` if `tree` is empty. sled uses the internal `DEFAULT_TREE_ID` for default db and we should not depend on this.

Reply:
> Do I need to consider that the old version of the sled service does not have this tree configuration and therefore cannot use the default value directly?

Yes, we need to make sure the behavior is the same as before while `tree` not specifed.

I don't know if it a good idea to use `DEFAULT_TREE_ID` directly:

https://github.com/spacejam/sled/blob/69294e59c718289ab3cb6bd03ac3b9e1e072a1e7/src/db.rs#L5

Reply:
I have used sled const DEFAULT_TREE_ID directly.